src/styles/index.scss

```diff
@@ -1,3 +1,66 @@
+@import './utils/variable';
```

Comment:
Ensure that the file './utils/variable' exists and is correctly imported. This file should define the SCSS variables used in this stylesheet.